### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by zsotroav**
**Description**

All changes made were based on Grammarly's suggestions. I didn't make big changes, only small grammar improvements (typos, missing commas, etc.)

### CODE REVIEW

File: _pages/en_US/move-emunand.txt [hunk 1]
```diff
@@ -54,10 +54,10 @@ Note that this is not necessary for any other kind of Virtual Console games (GBC
 The game will be outputted to the `/gm9/out/` folder on your SD card with the name `<TitleID>.gbavc.sav`.
 {: .notice--info}
 
-To identify a `<TitleID>.gbavc.sav` file's Title ID, you can get a listing of all games on the system and their corresponding Title IDs by hovering over `[A:] SYSNAND SD`, holding (R) and pressing (A) at the same time, then selecting "Search for titles".
+To identify a `<TitleID>.gbavc.sav` file's Title ID, you can get a listing of all games on the system and their corresponding Title IDs by hovering over `[A:] SYSNAND SD`, holding (R), and pressing (A) at the same time, then select "Search for titles".
```
Comment (aspargas2) [inline _pages/en_US/move-emunand.txt:57]: This is uh, wrong. It was fine like it was.

**Review by aspargas2 — CHANGES_REQUESTED** [review]
There's a few changes here which definitely are wrong.

File: _pages/en_US/ctrtransfer.txt [hunk 2]
```diff
@@ -63,7 +63,7 @@ Performing a CTRTransfer may break extended memory mode games (Monster Hunter, S
   + This prompt will only appear if you have an EmuNAND
 1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
   + This process will take some time
-1. Once the transfer has completed, press (A) to continue
+1. Once the transfer has been completed, press (A) to continue
```
Comment (lifehackerhansol) [inline _pages/en_US/ctrtransfer.txt:66]:
```suggestion
1. Once the transfer is complete, press (A) to continue
```

File: _pages/en_US/installing-boot9strap-(fredtool).txt [hunk 3]
```diff
@@ -48,7 +48,7 @@ If you see a configuration menu, you already have CFW and continuing with these
 1. Select "Verify files"
 1. Select "Build and Download"
 1. Wait for the process to complete
-1. When the process has completed, download your modified DSiWare archive from the site
+1. When the process has been completed, download your modified DSiWare archive from the site
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(fredtool).txt:51]:
```suggestion
1. When the process is complete, download your modified DSiWare archive from the site
```

File: _pages/en_US/installing-boot9strap-(kartdlphax).txt [hunk 4]
```diff
@@ -72,7 +72,7 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 
 1. Wait for all checks to complete
 1. When prompted, input the key combo given on the top screen to install boot9strap
-1. Once it has completed, press (A) to reboot your device
+1. Once it has been completed, press (A) to reboot your device
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(kartdlphax).txt:75]:
```suggestion
1. Once it is complete, press (A) to reboot your device
```

File: _pages/en_US/installing-boot9strap-(ntrboot).txt [hunk 5]
```diff
@@ -53,7 +53,7 @@ To use the [magnet](https://wikipedia.org/wiki/Magnet_URI_scheme) links on this
 1. Remove the magnet from your device
   + On old 2DS, you should instead disable the sleep mode switch
 1. When prompted, input the key combo given on the top screen to install boot9strap
-1. Once it has completed, force your device to power off by holding down the power button
+1. Once it has been completed, force your device to power off by holding down the power button
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(ntrboot).txt:56]:
```suggestion
1. Once it is complete, force your device to power off by holding down the power button
```

File: _pages/en_US/installing-boot9strap-(pichaxx).txt [hunk 6]
```diff
@@ -67,7 +67,7 @@ This method of using Seedminer for further exploitation uses your `movable.sed`
 
 1. Wait for all safety checks to complete
 1. When prompted, input the key combo given to install boot9strap
-1. Once it has completed, press (A) to reboot your device
+1. Once it has been completed, press (A) to reboot your device
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(pichaxx).txt:70]:
```suggestion
1. Once it is complete, press (A) to reboot your device
```

File: _pages/en_US/installing-boot9strap-(safeb9sinstaller).txt [hunk 7]
```diff
@@ -22,7 +22,7 @@ title: "Installing boot9strap (SafeB9SInstaller)"
 
 1. Wait for all safety checks to complete
 1. When prompted, input the key combo given to install boot9strap
-1. Once it has completed, press (A) to reboot your device
+1. Once it has been completed, press (A) to reboot your device
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(safeb9sinstaller).txt:25]:
```suggestion
1. Once it is complete, press (A) to reboot your device
```

File: _pages/en_US/installing-boot9strap-(soundhax).txt [hunk 8]
```diff
@@ -61,7 +61,7 @@ Soundhax (when combined with universal-otherapp) is compatible with versions 1.0
 
 1. Wait for all checks to complete
 1. When prompted, input the key combo given on the top screen to install boot9strap
-1. Once it has completed, press (A) to reboot your device
+1. Once it has been completed, press (A) to reboot your device
```
Comment (lifehackerhansol) [inline _pages/en_US/installing-boot9strap-(soundhax).txt:64]:
```suggestion
1. Once it is complete, press (A) to reboot your device
```

File: _pages/en_US/move-emunand.txt [hunk 9]
```diff
@@ -35,7 +35,7 @@ If you do not have any DSiWare games or saves that you care about, skip this sec
 {: .notice--info}
 
 1. Launch GodMode9 by holding (Start) during boot
-1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has completed
+1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has been completed
```
Comment (lifehackerhansol) [inline _pages/en_US/move-emunand.txt:38]:
```suggestion
1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it is complete
```

File: _pages/en_US/move-emunand.txt [hunk 10]
```diff
@@ -78,7 +78,7 @@ To identify a `<TitleID>.gbavc.sav` file's Title ID, you can get a listing of al
   + This will not overwrite your boot9strap installation
 1. Input the key combo given to unlock SysNAND (lvl1) writing
   + This process will take some time
-1. Once it has completed, press (A) to continue
+1. Once it has been completed, press (A) to continue
```
Comment (lifehackerhansol) [inline _pages/en_US/move-emunand.txt:81]:
```suggestion
1. Once it is complete, press (A) to continue
```

File: _pages/en_US/region-changing.txt [hunk 11]
```diff
@@ -96,7 +96,7 @@ Performing a CTRTransfer may break extended memory mode games (Monster Hunter, S
   + This prompt will only appear if you have an EmuNAND
 1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
   + This process will take some time
-1. Once the transfer has completed, press (A) to continue
+1. Once the transfer has been completed, press (A) to continue
```
Comment (lifehackerhansol) [inline _pages/en_US/region-changing.txt:99]:
```suggestion
1. Once the transfer is complete, press (A) to continue
```

File: _pages/en_US/region-changing.txt [hunk 12]
```diff
@@ -62,7 +62,7 @@ Performing a CTRTransfer may break extended memory mode games (Monster Hunter, S
 
 #### Section II - NAND Backup
 1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
-1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has completed
+1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has been completed
```
Comment (lifehackerhansol) [inline _pages/en_US/region-changing.txt:65]:
```suggestion
1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it is complete
```

File: _pages/en_US/seedminer.txt [hunk 13]
```diff
@@ -50,7 +50,7 @@ This method uses a powerful graphics card to perform the calculations needed. A
   + This is usually fast (1-5 minutes) but in some cases, can take up to half an hour
   + During this process, the bot you added may not show up on your 3DS. As long as the website updates, this is not an issue
   + If you are still waiting after half an hour, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask (in English) for someone there to assist you
-1. When the process has completed, download your `movable.sed` file from the site
+1. When the process has been completed, download your `movable.sed` file from the site
```
Comment (lifehackerhansol) [inline _pages/en_US/seedminer.txt:53]:
```suggestion
1. When the process is complete, download your `movable.sed` file from the site
```

File: _pages/en_US/uninstall-cfw.txt [hunk 14]
```diff
@@ -66,7 +66,7 @@ If either of these tests has failed, DS mode, DS Download Play, and/or DS Connec
 The purpose of this section is to verify that the console will boot and that critical system functions, like System Settings and the keyboard, will work once CFW is uninstalled. **If you skip this section, you may BRICK your device!**
 
 1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
-1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has completed
+1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has been completed
```
Comment (lifehackerhansol) [inline _pages/en_US/uninstall-cfw.txt:69]:
```suggestion
1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it is complete
```

File: _pages/en_US/updating-b9s.txt [hunk 15]
```diff
@@ -41,7 +41,7 @@ For all steps in this section, overwrite any existing files on your SD card.
   + This should automatically launch SafeB9SInstaller
 1. Wait for all safety checks to complete
 1. When prompted, input the key combo given to install boot9strap
-1. Once it has completed, force your device to power off by holding down the power button
+1. Once it has been completed, force your device to power off by holding down the power button
```
Comment (lifehackerhansol) [inline _pages/en_US/updating-b9s.txt:44]:
```suggestion
1. Once it is complete, force your device to power off by holding down the power button
```

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

File: _pages/en_US/troubleshooting.txt [hunk 16]
```diff
@@ -10,7 +10,7 @@ This page offers troubleshooting advice for commonly encountered issues. If you
 
 ## Issues with SafeB9SInstaller
 
-### SigHaxed FIRM was not installed! Check lower screen for more info.
+### SigHaxed FIRM was not installed! Check the lower screen for more info.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:13]: This should not be changed as it is based on the message that shows up on the 3DS itself

**Review by lilyuwuu — COMMENTED** [review]
Everything else looks good at a glance

File: _pages/en_US/uninstall-cfw.txt (hunk 14 above)
Comment (lifehackerhansol) [reply _pages/en_US/uninstall-cfw.txt:69]: This was also not added lol

File: _pages/en_US/seedminer.txt (hunk 13 above)
Comment (lifehackerhansol) [reply _pages/en_US/seedminer.txt:53]: This was not added.

File: _pages/en_US/region-changing.txt (hunk 11 above)
Comment (lifehackerhansol) [reply _pages/en_US/region-changing.txt:99]: This was not added.

File: _pages/en_US/region-changing.txt (hunk 12 above)
Comment (lifehackerhansol) [reply _pages/en_US/region-changing.txt:65]: This was not added.

File: _pages/en_US/move-emunand.txt (hunk 9 above)
Comment (lifehackerhansol) [reply _pages/en_US/move-emunand.txt:38]: This was not added.

File: _pages/en_US/move-emunand.txt (hunk 10 above)
Comment (lifehackerhansol) [reply _pages/en_US/move-emunand.txt:81]: This was not added.

File: _pages/en_US/installing-boot9strap-(ntrboot).txt (hunk 5 above)
Comment (lifehackerhansol) [reply _pages/en_US/installing-boot9strap-(ntrboot).txt:56]: This was not added.

File: _pages/en_US/installing-boot9strap-(fredtool).txt (hunk 3 above)
Comment (lifehackerhansol) [reply _pages/en_US/installing-boot9strap-(fredtool).txt:51]: This was not added.

File: _pages/en_US/ctrtransfer.txt (hunk 2 above)
Comment (lifehackerhansol) [reply _pages/en_US/ctrtransfer.txt:66]: This was not added.

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

**Review by lifehackerhansol — APPROVED** [review]

**Pull request merged**